docroot/profiles/custom/cgov_site/modules/custom/cgov_syndication/src/Plugin/Field/FieldWidget/CgovSyndicationWidget.php

```diff
@@ -45,6 +45,7 @@ public function formElement(
       FormStateInterface $form_state
   ) {
     $item = $items[$delta];
+    /* @phpstan-ignore-next-line */
     $open = $item->syndicate || !empty($item->keywords);
```

Comment:
Can you do this without needing to ignore the next line?

Reply:
Resolved.